agentdock-core/src/evaluation/evaluators/rule-based/evaluator.ts

```
@@ -0,0 +1,237 @@
+            rulePassed = this.evaluateRegex(textToEvaluate!, rule.config); // textToEvaluate ensured by check above
+            break;
+          case 'length':
+            rulePassed = this.evaluateLength(textToEvaluate!, rule.config); // textToEvaluate ensured by check above
+            break;
+          case 'includes':
+            rulePassed = this.evaluateIncludes(textToEvaluate!, rule.config); // textToEvaluate ensured by check above
+            break;
+          case 'json_parse':
+            // For json_parse, if input.response is a string, use it directly.
+            // Otherwise, stringify the raw input.response object.
+            const stringToParse = typeof input.response === 'string' ? input.response : JSON.stringify(input.response);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)